Jenkinsfile

```
@@ -13,11 +13,11 @@ properties([
 // Define the sequential stages and the parallel steps inside each stage
 def sequentialStages = [:]
 sequentialStages['Tool'] = [ 'java', 'maven', 'maven-11', 'maven-windows', 'maven-11-windows']
-sequentialStages['OS & Java'] = [ 'linux', 'jdk8', 'jdk11', 'windows']
+sequentialStages['OS & Java'] = [ 'linux', 'jdk8', 'jdk11', 'jdk17'] // Remove windows label as removed and add jdk17 as used
```

```suggestion
sequentialStages['OS & Java'] = [ 'linux', 'jdk8', 'jdk11', 'jdk17'] // Label definition found here: https://github.com/jenkins-infra/pipeline-library/blob/master/vars/buildPlugin.groovy#L45-L65
```

not sure to understand why adding the comment and its meaning, so I propose to comment with the link to the label definition.

---

Second though: I'm not sure if it make sense to test for the labels `jdk8`, etc. 
What are the pipeline or library consuming these labels? (question for both @smerle33 and @MarkEWaite )

---

> Second though: I'm not sure if it make sense to test for the labels `jdk8`, etc. What are the pipeline or library consuming these labels? (question for both @smerle33 and @MarkEWaite )

I don't see any reference to jdk8 within pipeline-library but not sure that it's the main reference. Shouldn't we use the ci logs to ensure the list for agent currently used ?
from the code in pipeline-library I can only see : 
- maven-11
- docker-windows
- docker || linux-amd64-docker
- maven
- maven-(+jdk)(+-windows) map
- vm && linux
- linux

not quite sure for :
- jnlp

---

we have to search the whole pipeline-library repository, but also the whole jenkinsci and jenkins-infra.

I totally challenge the fact that these labels (`jdk8`, `jdk11`, `jdk17`) are tested unless we know for sure where there are used.

----

`jnlp` could have been a former "default". I don't think we should test it either (to be searched as well).